mubench.reviewsite/tests/MisuseTest.php

```diff
@@ -34,4 +41,71 @@ public function testNoResolutionReview()
 
         self::assertFalse($misuse->hasResolutionReview());
     }
+
+    public function testCheckSnippetsWithFindings()
+    {
+        $this->createRunWithFinding(11);
+
+        SnippetsController::createSnippetIfNotExists("-p-", "-v-", "-m-", "//src/file", 10, "m(A){\n\ta(A);\n}");
+
+        $run = Run::of(Detector::find('-d-'))->in(Experiment::find(1))->first();
+
+        self::assertEquals(1, $run->misuses->count());
+        self::assertEquals(1, count($run->misuses[0]->snippets()));
+        self::assertEquals(10, $run->misuses[0]->snippets()[0]->line);
+    }
+
+    public function testCheckSnippetsWithMetadataAndFindings()
```

Comment:
`testFiltersUnrelatedSnippet()`, should have one finding and two snippets, whereof one does not fit due to line numbers check. Metadata is irrelevant here.